shopinvader_restapi/views/shopinvader_menu.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2017 Akretion
+    Copyright 2021 Camptocamp (http://www.camptocamp.com).
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+    <menuitem
+        id="menu_shopinvader_root"
+        name="Shopinvader"
+        sequence="10"
+        web_icon="shopinvader,static/description/icon.png"
```

Comment:
This path must be changed since `shopinvader_restapi` does not depends on `shopinvader` anymore